UNOMI-292 Implement cdp_listsUpdateEvent & processing

pmi opened the pull request.

anatol-sialitski commented on graphql/cxs-impl/src/main/java/org/apache/unomi/graphql/condition/EventConditionFactory.java:
```diff
@@ -84,4 +89,21 @@ public Condition createEventFilterInputCondition(CDPEventFilterInput filterInput
+
+        final List<Condition> rootSubConditions = new ArrayList<>();
+
+        if (cdp_listsUpdateEvent.getJoinLists_contains() != null && cdp_listsUpdateEvent.getJoinLists_contains().size() > 0) {
```
`!isEmpty()` instead of `size() > 0`

pmi committed:
UNOMI-292 Implement cdp_listsUpdateEvent & processing

- fixed CustomObjectMapper to deserialize Profile
- also fixed ProfileListDataFetcher to load only existing user lists
- refactored CDPEvent and CDPList to do data loading in getters instead of constructor
